Orbitersdk/samples/ProjectApollo/src_sys/apolloguidance.cpp

```diff
@@ -836,41 +836,31 @@ void ApolloGuidance::ProcessIMUCDUReadCount(int channel, int val) {
 }
 
 void ApolloGuidance::GenerateHandrupt() {
-	GenerateHANDRUPT(&vagc);
+	vagc.InterruptRequests[10] = 1;
```

Comment:
So basically you're just copying the bodies of these functions out of agc_engine.c, so you can later delete the functions from there? Seems reasonable.